src/subscriptions.rs

```diff
@@ -241,7 +281,20 @@ impl SubscriptionsHandler {
 
         match response {
             Ok(res) => debug!("Sent message with body '{}' and created message_id {}", msg_str, res.message_id.unwrap()),
-            Err(error) => debug!("SendMessageError: {:?}", error),
+            Err(error) => {
```

Comment:
What does this pattern matching give?
It seems like the same handling before, only that `debug!` turned into `info!` + the handling of `Unknown` case a bit differently.

Reply:
Yes, expanded to handle the error for the unknown case so we dont see a raw buffer

Reply:
Ok. So maybe just create a helper function to remove all the duplicate logs?
e.g. 
```
fn logSendError(err_msg: &str) {
  info!("Error while sending message {}", err_msg)
} 
```
and have all the different branches to call it?...